flips/20220203-capability-controllers.md

```diff
@@ -0,0 +1,390 @@
+# Capability controllers
+
+| Status        | Proposed                                           |
+| :------------ | :------------------------------------------------- |
+| **FLIP #**    | [798](https://github.com/onflow/flow/pull/798)     |
+| **Author(s)** | Janez Podhostnik (janez.podhostnik@dapperlabs.com) |
+| **Updated**   | 2022-08-22                                         |
```

Comment:
```suggestion
 status: Proposed
| flip: [798](https://github.com/onflow/flow/pull/798)
| author: Janez Podhostnik (janez.podhostnik@dapperlabs.com)
| updated: 2022-08-22

# Capability controllers
```

Reply:
This info section format should be that of Frontmatter, so that it can be parsed by markdown parses and used for automation, e.g. it can be included in the docs site.

Reply:
addressed